jrodriguezdiazz/book-search-engine
# Add book search functionality from the Google API

## Description

Add an input that allows the user to search for the name of a book using the Google Books API



---

## Motivation and Context

The search field is the main tool of the project so it was very important to add this feature





---

## How Has This Been Tested?

1. Go to the next link: https://book-search-engine-j9s2fo7gv-jrodriguezdiazz.vercel.app/
2. Type the name of any book in the input
3. Open the Chrome DevTools with `Control + Shift + I`
4. Go to the Network tab
5. Verify if it has one request to the Google Books API
6. Check the name of the books 






---

## Screenshots (if appropriate):

| Desktop      | Mobile |
| ----------- | ----------- |
| ![image](https://user-images.githubusercontent.com/24766287/144538611-24bee59e-8dc6-4d1b-8085-801b83ddbde6.png) | ![image](https://user-images.githubusercontent.com/24766287/144538763-e56cc31f-22fc-4dd8-8672-6f97b136cfe8.png) |



---

## Types of Changes




---

## Checklist:





---

# Other Notes


At the moment we have hardcoded test data for the input to work, later they will be eliminated
